crates/astria-composer/src/config/mod.rs

```diff
@@ -0,0 +1,190 @@
+use clap::Parser;
+use figment::{
+    map,
+    providers::{
+        Env,
+        Serialized,
+    },
+    Figment,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+mod cli;
+pub mod searcher;
+
+// TODO: add more default values
+// potentially move to a separate module so it can be imported into searcher and block_builder?
+const DEFAULT_LOG: &str = "info";
+
+fn default_log() -> String {
+    DEFAULT_LOG.into()
+}
+
+#[derive(Debug, Deserialize, Serialize, PartialEq)]
+/// The high-level config for creating an astria-composer service.
+pub struct Config {
+    /// Log level. One of debug, info, warn, or error
+    #[serde(default = "default_log")]
+    pub log: String,
```

Comment:
do we need a log option or can we just go wtth ` RUST_LOG`?

Reply:
good point